### PR TITLE
jsoncpp: update to 1.9.5

### PR DESCRIPTION
**Pull request by MehdiChinoune**

### CODE REVIEW

**Comment by lazka** [comment]
ABI break, needs rebuilds

**Comment by MehdiChinoune** [comment]
- Building CMake with CMake is not reliable
- Again with CMake picking libglut32 over freeglut on 32bit environments, First was partio, Second was mathgl and now vrpn.
  As I said before Either get rid of libglut32 or we patch CMake to ignore it.

File: mingw-w64-cmake/PKGBUILD [hunk 1]
```diff
@@ -99,10 +102,11 @@ build() {
     MSYSTEM=MINGW FC=${MINGW_PREFIX}/bin/gfortran.exe \
       "${srcdir}"/${_realname}-${pkgver}/configure  \
       --prefix=${MINGW_PREFIX}                      \
+      --datadir=/share/cmake                        \
+      --docdir=/share/doc/cmake                     \
+      --mandir=/share/man                           \
```
Comment (jeremyd2019) [inline mingw-w64-cmake/PKGBUILD:107]: I'm concerned with these looking like 'absolute paths' now.  Were they not working as 'relative' paths (at least the ones that already existed below)?

Comment (MehdiChinoune) [reply mingw-w64-cmake/PKGBUILD]: I have just copied it from ArchLinux.

Comment (jeremyd2019) [reply mingw-w64-cmake/PKGBUILD]: would probably have been ok in a configure script, but my concern was that some native Windows executable would get them at some point, and the msys2 arg conversion would try to turn them into absolute Windows paths.  As long as the 'relative' form works, no reason to take the risk.

**Comment by MehdiChinoune** [comment]
ready

**Comment by MehdiChinoune** [comment]
ready

**Pull request closed without merging**